code/lib/types/src/modules/addons.ts

```diff
@@ -339,6 +339,39 @@ interface FCWithoutChildren<P = {}> {
   displayName?: string | undefined;
 }
 
+export interface Addon_PageType {
+  type: Addon_TypesEnum.experimental_PAGE;
+  /**
+   * The unique id of the page.
+   */
+  id: string;
+  /**
+   * The URL to navigate to when Storybook needs to navigate to this page.
+   */
+  url: string;
```

Comment:
Don't we want to support a URL scheme? Like I am imagining we'll want to have something like `/visual-test/$storyId` (or similar containing `$storyId`). Perhaps what I am saying is that it should maybe be possible to add a view mode?